Bottom: bab4ea2a0cb8d6ea91fb0b60a83fcdda27314454
Top:    e50f65b717e92423e8fb8deb12176c88c0b87ff6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 15:29:01 +0000

Avoid clearing pins / vias from polygons... it is already done!











---

diff --git a/src/draw.c b/src/draw.c
index 1ccd1e6..7a506d0 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1291,6 +1291,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1313,6 +1314,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
